tests/scaffoldings/settings/docker-compose.yaml

```diff
@@ -17,6 +17,7 @@ services:
       PGHOST: postgresql
       PGPORT: 15432
       PGDATABASE: ${DBNAME:-prod}
+      DBFILTER: ${DBFILTER:-.*}
```

Comment:
This is just the default value, so you will not see the setting effect.

You can remove this line. However, it would be great if you can expand [`test_settings()`](https://github.com/Tecnativa/docker-odoo-base/blob/8bfba4faa9fc07a231b45bc48553ba037c589198/tests/__init__.py#L165) to make sure odoo gets and processes this setting correctly.

Reply:
Honestly I don't really know how to do it :sweat_smile:

Reply:
I did a tentative commit, probably not in the right place, but, should the assertion be done in a similar script?